### PR TITLE
Fix jetson (non container) build

### PR DESCRIPTION
**Pull request by CoderHam**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -45,11 +47,24 @@ if(NOT CMAKE_BUILD_TYPE)
...
+endif()
+
+if (${TRITON_TENSORFLOW_DOCKER_BUILD})
+  set(TRITON_TENSORFLOW_LIBNAME "libtensorflow_trtis.so")
+else()
+  set(TRITON_TENSORFLOW_LDFLAGS "-ltensorflow_trtis")	
+endif() # TRITON_TENSORFLOW_DOCKER_BUILD
 set(TRITON_TENSORFLOW_BACKEND_LIBNAME triton_tensorflow1)
 set(TRITON_TENSORFLOW_BACKEND_INSTALLDIR ${CMAKE_INSTALL_PREFIX}/backends/tensorflow1)
 if (${TRITON_TENSORFLOW_VERSION} EQUAL "2")
```
Comment (deadeyegoodwin) [inline CMakeLists.txt:62]: Can you nest the "2" checks into the above conditional to keep the related code together.. at lines 56 and 58.

File: CMakeLists.txt [hunk 2]
```diff
@@ -138,9 +155,15 @@ add_library(
 
 target_include_directories(
   triton-tensorflow-backend
-  PRIVATE
-    ${CMAKE_CURRENT_SOURCE_DIR}/src
+  PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/src
+)
+
+if (NOT ${TRITON_TENSORFLOW_DOCKER_BUILD})
+target_include_directories(
```
Comment (deadeyegoodwin) [inline CMakeLists.txt:162]: indent

File: CMakeLists.txt [hunk 3]
```diff
@@ -36,6 +36,8 @@ option(TRITON_ENABLE_STATS "Include statistics collections in backend" ON)
 set(TRITON_TENSORFLOW_VERSION "1" CACHE STRING "TensorFlow version, must be '1' or '2'")
 set(TRITON_TENSORFLOW_DOCKER_IMAGE "nvcr.io/nvidia/tensorflow:20.08-tf1-py3" CACHE STRING
     "Docker image containing the TensorFlow build required by backend")
+set(TRITON_TENSORFLOW_LIB_PATHS "" CACHE PATH "Paths to TensorFlow libraries")
```
Comment (deadeyegoodwin) [inline CMakeLists.txt:39]: Add "Multiple paths may be separated with a semicolon." to description of both of these new vars.

**Review by deadeyegoodwin — CHANGES_REQUESTED** [review]

File: CMakeLists.txt (hunk 3 above)
Comment (CoderHam) [reply CMakeLists.txt]: Done

File: CMakeLists.txt (hunk 1 above)
Comment (CoderHam) [reply CMakeLists.txt:57]: Done.

**Review by deadeyegoodwin — DISMISSED** [review]

**Pull request merged**